staging/src/k8s.io/api/testdata/HEAD/batch.v1.Job.yaml

```diff
@@ -43,6 +43,7 @@ spec:
       - 3_bQw.-dG6c-.x
     matchLabels:
       hjT9s-j41-0-6p-JFHn7y-74.-0MUORQQ.N4: 3L.u
+  suspend: true
```

Comment:
I guess I expected only this change and not the others in these golden data files?

Reply:
I didn't expect this to change either, but these testdata changes are automatically generated by `UPDATE_COMPATIBILITY_FIXTURE_DATA=true go test -run=TestCompatibility`. FWIW Aldo's PR's codegen (https://github.com/kubernetes/kubernetes/pull/98812) also generated identical changes to these files.

Reply:
OK